### ISSUE TITLE
Fingerprinting enabled 

### ISSUE THREAD

**Issue by mylastore** [nodeca/pica#244]
Is there a method to enable Pica to function even when fingerprint protection is enabled?

Error: Pica: cannot use getImageData on canvas, make sure fingerprinting protection isn't enabled
    at pica.js:2480:17
    at Array.<anonymous> (lie.min.js:1:2114)

**Comment by puzrin** [comment]
No, sorry.

**Comment by NoelDeMartin** [comment]
In case anyone else comes across this, it can actually be enabled by replacing this if in `lib/utils.js` with `usable = true`:

```js
if (d.data[0] === 12 && d.data[1] === 23 && d.data[2] === 34 && d.data[3] === 255 &&
    d.data[4] === 45 && d.data[5] === 56 && d.data[6] === 67 && d.data[7] === 255) {
  usable = true;
}
```

It seems like there is also a PR open with a better fix: #239

It is true that in some environments, such as Firefox private mode, the canvas is modified to prevent fingerprinting. However, that doesn't mean the canvas is completely unusable, given that most of these modifications are impercetible by humans. Depending on your use-case, you'll want to use this library anyways.

If you want to apply any of these fixes to your project, you can do it without a new `pica` release. Just use something like [patch-package](https://www.npmjs.com/package/patch-package) to apply the fixes in your project (or fork the repository if you need the library in multiple projects, etc.).
